npm/react/scripts/createDistPackageJson.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs')
+const path = require('path')
+const mainPackageData = require('../package.json')
+
+function run () {
+  const distPackageData = JSON.stringify({
```

Comment:
It's an overall useful practice to not publish the whole project but only `dist` path. 
We can consider to also publish package using `yarn publish dist` in the future.